recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -413,6 +412,15 @@ install:
               fi
             fi
 
+            # Adding a quick test to ensure we don't expose any keys in case non-compatible versions of sed are used
+            SEDTEST=$(echo 0 | sed -E 's/[a-zA-Z0-9\-]/1/')
+
+            # Add helm upgrade command as metadata.  Keys are obfuscated.
+            if [[ $SEDTEST -eq 1 ]]; then
+              CLEANED_ARGS=$(echo $ARGS | sed -E 's/licenseKey=[a-zA-Z0-9\-]+ ?/licenseKey=XXXXX /g' | sed -E 's/deployKey=[a-zA-Z0-9\-]+ ?/deployKey=XXXXX /g' | sed -E 's/apiKey=[a-zA-Z0-9\-]+ ?/apiKey=XXXXX /g')
```

Comment:
I think you should remove the space after `[a-zA-Z0-9\-]`. If for some reason the key is in the last argument, it would not be obfuscated if you include the space.

Reply:
Good point, updated.